docs/Getting-Started.md

```diff
@@ -56,61 +44,167 @@ fastify.listen(3000, function (err) {
 })
 ```
 
-*To learn more about using serialization and validation, see [Validation and Serialization](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md)!*
+Awesome, that was easy.  
+Unfortunately, write a complex application requires definitely more code than what we wrote above. A classic problem when you are building a new application is how handle multiple files, asynchronous bootstrapping and the architecture of your code.  
+Fastify offers you out of the box an easy to use system that will help you handle all this problems and more.
 
-<a name="register"></a>
-### Register
-As you can see, using Fastify is very easy and handy. Obviously, registering all of your routes in the same file is not a good idea, so Fastify offers a utility to solve this problem, `register`:
+<a name="first-plugin"></a>
+### Your first plugin
+As in javascript everything is an object, in Fastify everything is a plugin.  
+Before digging into it, let's see how it works!  
+Let's declare our basic server, but instead of declare the route inside the entry point, we'll declare it in an external file (checkout the [route declaration](https://github.com/fastify/fastify/blob/master/docs/Routes.md) docs).
```

Comment:
> Let's declare our basic server, but instead of ~declare~ declaring the route inside the entry point, we'll declare it in an external file (checkout the [route declaration](https://github.com/fastify/fastify/blob/master/docs/Routes.md) docs).